components/create_post/create_post.jsx

```diff
@@ -363,15 +363,10 @@ export default class CreatePost extends React.Component {
         post.parent_id = this.state.parentId;
 
         GlobalActions.emitUserPostedEvent(post);
-        this.props.actions.createPost(post, this.props.draft.fileInfos).then(() => GlobalActions.postListScrollChange(true)).catch((err) => {
```

Comment:
What happened to `GlobalActions.postListScrollChange(true)`? I don't see it being called anywhere

Reply:
I missed that. PR updated.
Thanks!